.eslintrc

```diff
@@ -0,0 +1,29 @@
+{
+    "globals": {
+        // Put things like jQuery, etc
+        "jQuery": true,
+        "$": true
```

Comment:
what's the difference between `jQuery` & `$`? If they are the same, why are you using both interchangeably? 

also, why not enable `jquery: true` in the `env`? (like what you are doing for `browser` & `node`